.github/workflows/pullrequest.yml

```
@@ -276,6 +284,21 @@ jobs:
       - name: Building
         run: ./scripts/ci/run-in-parallel-native.sh build
 
+  docker-build:
+    uses: ./.github/workflows/docker-build.yml
+    needs:
+      - pre-checks
+    secrets: inherit
+    with:
+      GIT_BRANCH: ${{ needs.pre-checks.outputs.GIT_BRANCH}}
+      DOCKER_TAG: ${{ needs.prepare.outputs.DOCKER_TAG}}
```

actionlint (1.7.4)

property "prepare" is not defined in object type {pre-checks: {outputs: {cache_key: string; feature_name: string; git_base_branch: string; git_branch: string; git_branch_deploy: string; node_image_version: string; pre_check: string; pre_release: string}; result: string}}

(expression)